### PR TITLE
fix: label changes in Account Settings

### PR DESCRIPTION
**Pull request by 0925-pinka**
Issue: https://corp.bloomstack.com/desk#Form/Task/TASK-2020-00159

Before:

Screenshot from 2020-03-17 10-35-03

After:

Screenshot from 2020-03-17 10-39-19

### CODE REVIEW

**Comment by vjFaLk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- erpnext/accounts/doctype/accounts_settings/regional/united_states.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DigiThinkIT/erpnext/pullRequest?prid=5192234&bid=17031140)

**Comment by Alchez** [comment]
@pinka0925 Can you please the add the task link and fix all the whitespace issues?

File: erpnext/regional/united_states/setup.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def setup(company=None, patch=True):
 
 
 def make_custom_fields():
-	custom_fields = {
+    	custom_fields = {
```
Comment (Alchez) [inline erpnext/regional/united_states/setup.py]: Remove all changes from this file

File: erpnext/accounts/doctype/accounts_settings/regional/united_states.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+frappe.ui.form.on('Accounts Settings', {
+	refresh: function(frm) {
+		frm.set_df_property("acc_frozen_upto","label","Books Closed Through"),
+		frm.set_df_property("frozen_accounts_modifier","label","Role  Allowed to Close Books & Make Changes to Closed Periods"),
+		frm.set_df_property("credit_controller","label","Credit Manager")
```
Comment (Alchez) [inline erpnext/accounts/doctype/accounts_settings/regional/united_states.js]: Add spaces between arguments

```suggestion
		frm.set_df_property("acc_frozen_upto", "label", "Books Closed Through");
		frm.set_df_property("frozen_accounts_modifier", "label", "Role Allowed to Close Books & Make Changes to Closed Periods");
		frm.set_df_property("credit_controller", "label", "Credit Manager");
```

**Review by Alchez — CHANGES_REQUESTED** [review]

**Comment by vjFaLk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- erpnext/accounts/doctype/accounts_settings/regional/united_states.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DigiThinkIT/erpnext/pullRequest?prid=5192234&bid=17031140)

**Comment by vjFaLk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- erpnext/accounts/doctype/accounts_settings/regional/united_states.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DigiThinkIT/erpnext/pullRequest?prid=5192234&bid=17031140)

**Comment by vjFaLk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- erpnext/accounts/doctype/accounts_settings/regional/united_states.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DigiThinkIT/erpnext/pullRequest?prid=5192234&bid=17031140)

**Review by Alchez — APPROVED** [review]

**Comment by Alchez** [comment]
@pinka0925, looks good. Can you make this upstream to develop as well?

**Comment by 0925-pinka** [comment]
Yeah, sure

**Comment by Alchez** [comment]
Made upstream to [hotfix](https://github.com/frappe/erpnext/pull/20964) and [develop](https://github.com/frappe/erpnext/pull/20965).

**Pull request merged**